index.html

```diff
@@ -94,32 +94,40 @@ <h2 class="header">Things we've done</h2>
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" alt="Two custom arcade machines in the CSH library" />
-                        </picture>
+                        <a href="https://github.com/ComputerScienceHouse/Arcade-Controls">
+                            <picture>
+                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.webp" type="image/webp" />
+                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" type="image/jpeg" />
+                                <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/arcade.jpg" alt="Two custom arcade machines in the CSH library" />
+                            </picture>
+                        </a>
                     </div>
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" alt="Two of CSH's networked vending machines, Drink and Snack.">
-                        </picture>
+                        <a href="https://github.com/ComputerScienceHouse/WebDrink-2.0">
+                            <picture>
+                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.webp" type="image/webp" />
+                                <source srcset="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" type="image/jpeg" />
+                                <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/drink.jpg" alt="Two of CSH's networked vending machines, Drink and Snack.">
+                            </picture>
+                        </a>
                     </div>
                     <div class="col-3">
-                        <picture>
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/harold.webp" type="image/webp" />
-                            <source srcset="https://assets.csh.rit.edu/pubsite/projects/harold.jpg" type="image/jpeg" />
-                            <img class="rounded"  src="https://assets.csh.rit.edu/pubsite/projects/harold.jpg" alt="Harold, a box with a sensor on it, that plays members' music" />
-                        </picture>
+                        <a href="https://github.com/ComputerScienceHouse/Harold">
```

Comment:
Similarly, I thought there was a new implementation of HAROLD being made (by Nate maybe?)

Reply:
It's not a CSH repo